backend/cf_arena/arena/views.py

```diff
@@ -1,10 +1,11 @@
 import requests
 from rest_framework.views import APIView
 from rest_framework.response import Response
-
 import constants
-from .helpers import *
-from .models import *
+import math
+import random
+from .helpers import generate_problem_url
```

Comment:
This will break the existing code. In helpers, there is a function name `helper_reponse`, can you import that please?